holoviews/plotting/widgets/jsslider.jinja

```diff
@@ -102,7 +102,8 @@
                                     } else {
                                         var dim_val = vals[ui.value];
                                     }
-                                    $('#textInput{{ id }}_{{ widget_data['dim'] }}').val(dim_val);
+									console.log((dim_val*1).toString())
```

Comment:
Is this a stray ``console.log`` or a useful one? If it is useful, I assume this won't be called too often.